spec/jest_spec.vim

```diff
@@ -107,7 +107,7 @@ describe "Jest"
     view outside-test.js
     TestFile
 
-    Expect g:test#last_command == 'jest outside-test.js'
+    Expect g:test#last_command == 'jest -- outside-test.js'
```

Comment:
In this case wouldn't we just want it to be `jest outside-test.js` as per the official documentation?